compiler/main/driver.cpp

```diff
@@ -1065,6 +1066,7 @@ static ArgumentDescription arg_desc[] = {
  {"library-fortran-name", ' ', "<modulename>", "Name generated Fortran module", "P", fortranModulename, NULL, setFortranAndLibmode},
  {"library-python", ' ', NULL, "Generate a module compatible with Python", "F", &fLibraryPython, NULL, setLibmode},
  {"library-python-name", ' ', "<filename>", "Name generated Python module", "P", pythonModulename, NULL, setPythonAndLibmode},
+ {"library-ml-debug", ' ', NULL, "Enable [disable] generation of debug messages in multi-locale libraries", "N", &fMultiLocaleLibraryDebug, NULL},
```

Comment:
It would be good to generate a warning if this flag is thrown when we aren't generating multilocale interoperable libraries.  I think you could write a test to validate that today, too.

Reply:
Never even thought about that. Ok!

Reply:
That's the only thing I'd want from this PR that it doesn't currently do (and the full paratest :p)